src/index.scss

```diff
@@ -162,31 +151,33 @@ html, body, #root, #app, .full-height{
 
   border-style: solid;
   border-width: 1px;
-  border-color: var(--dullblue);
+  border-color: $blue;
   border-radius: 15px;
 
-  background-color: var(--white);
+  background-color: $white;
   background-image: url(./input-arrow.svg);
   background-repeat: no-repeat;
   background-position: 3.75px;
   background-size: 18px;
 
   outline: none;
-  transition: 0.1s;
-} /* arrow moves slightly to the right on hover */
-.mailchimp-button:hover{
-  background-position: 5.25px;
-  transition: 0.1s;
-}
+  transition: 0.1s; /* see: min-width: media query */
+} 
 
 ::placeholder{ /* this refers to the input placeholder text (enter your email...)*/
```

Comment:
id suggest putting this under the class like
`.mailchimp-input::placeholder {...}`

Reply:
Actually -- nest this under the existing `.mailchimp-input` like dis:
```css
.mailchimp-input {
    ...
    &::placeholder {
        ...
    }
}
```